rohmu/notifier/interface.py

```diff
@@ -0,0 +1,32 @@
+"""Copyright (c) 2022 Aiven, Helsinki, Finland. https://aiven.io/"""
+from abc import ABC, abstractmethod
+from typing import Optional
+
+
+class Notifier(ABC):
+    """This interface allows external code to be notified about object changes."""
+
+    @abstractmethod
+    def object_created(self, key: str, size: Optional[int]) -> None:
```

Comment:
nit: missing docstring on this one compared to the others?

```suggestion
    def object_created(self, key: str, size: Optional[int]) -> None:
    """Called when and object is created."""
```

Reply:
Added as per the request. Note that I don't think this is improving things, since the comment says the same thing that is expressed by the method name. The other methods had docstrings because it required a note to express its behavior.

Reply:
of course I managed to slip in a typo in half a sentence :'(  `and` -> `an` sorry :-/

Yeah in this case the comment doesn't look like it provides much benefit, except maybe consistency, but down the line if we generate source docs it will be a tad better.